src/core/FileWatcher.cpp

```diff
@@ -135,7 +135,7 @@ void BulkFileWatcher::clear()
     }
     m_watchedPaths.clear();
     m_watchedFilesInDirectory.clear();
-    m_ignoreFilesChangess.clear();
+    m_ignoreFilesChanges.clear();
```

Comment:
Can we just rename this to m_ignoredFiles?

Reply:
I think the current name is better since we want to ignore changes to the files for a certain time instead of ignoring them completely.

Reply:
At least name it `m_ignoreFileChanges` then. The double plurals gives me the willies. ;-)

Reply:
I wouldn't take the name too literally. Right now the name is a brain bender and really make no sense whatsoever.

Reply:
I like your suggestion. I renamed `m_ignoreFilesChanges`.